mimic/model/heat_objects.py

```diff
@@ -135,6 +135,8 @@ def creation_response_json(self, absolutize_url):
 
 
 stack_creation = EventDescription()
+stack_check = EventDescription()
```

Comment:
Honestly, after seeing how sparingly EventDescriptions are used elsewhere in mimic and that default behaviors are the only ones used, I think that refactoring them out of the heat mimic would be better.